apis/src/components/molecules/tournament_finished_notification.rs

```diff
@@ -0,0 +1,41 @@
+use crate::providers::NotificationContext;
+use crate::responses::TournamentResponse;
+use leptos::*;
+use leptos_icons::*;
+
+#[component]
+pub fn TournamentFinishedNotification(
+    tournament: StoredValue<TournamentResponse>,
+) -> impl IntoView {
+    let div_class = "xs:py-1 xs:px-1 sm:py-2 sm:px-2";
+
+    let dismiss = move |_| {
+        let notifications = expect_context::<NotificationContext>();
+        notifications.tournament_finished.update(|t| {
+            t.remove(&tournament().tournament_id);
+        });
+    };
+
+    view! {
+        <div class="flex items-center text-center cursor-pointer dark:odd:bg-header-twilight dark:even:bg-reserve-twilight odd:bg-odd-light even:bg-even-light max-w-fit">
+            <div class="flex relative">
+                <div class=div_class>
+                    <div>{tournament().name} " finished"</div>
```

Comment:
Probably another line of text saying "Tournament notification" would be nicer, just seeing "X finished might be a bit confusing